engines/sci/sci.cpp

```diff
@@ -436,7 +436,21 @@ Common::Error SciEngine::run() {
 	}
 
 	if (getGameId() == GID_GK2 && ConfMan.getBool("subtitles") && !_resMan->testResource(ResourceId(kResourceTypeSync, 10))) {
-		suggestDownloadGK2SubTitlesPatch();
+		suggestDownloadPatch(_("GK2 has a fan made subtitles, available thanks to the good persons at SierraHelp.\n\n"
+			"Installation:\n"
+			"- download http://www.sierrahelp.com/Files/Patches/GabrielKnight/GK2Subtitles.zip\n"
+			"- extract zip file\n"
+			"- no need to run the .exe file\n"
+			"- extract the .exe file with a file archiver, like 7-zip\n"
```

Comment:
[According to the official website](https://www.7-zip.org), it's "7-Zip", not "7-zip". I think that should be corrected. Also, isn't it a "file unarchiver" instead of "archiver" in this case, since we're extracting not packing?